spring-web/src/main/java/org/springframework/web/bind/UnsatisfiedServletRequestParameterException.java

```diff
@@ -49,7 +48,7 @@
 	 */
 	public UnsatisfiedServletRequestParameterException(String[] paramConditions, Map<String, String[]> actualParams) {
 		super("");
-		this.paramConditions = Arrays.<String[]>asList(paramConditions);
+		this.paramConditions = Collections.singletonList(paramConditions);
```

Comment:
What problem can this solve? And there is one more constructor, doesn't it need immutable handling?

Reply:
oh no is that my technology is not good, maybe this problem should be replaced by optimization instead of repair。